samples/drivers/gnss/README.rst

```diff
@@ -0,0 +1,28 @@
+.. _gnss_sample:
+
+GNSS sample
+###########
```

Comment:
use new sample directive (note that this won't make a difference for now has there's no documentation page showing the reference API).

```suggestion
.. zephyr:code-sample:: gnss
   :name: GNSS
   :relevant-api: gnss_interface navigation

   Connect to a GNSS device to obtain time, navigation data, and satellite information.
```

Reply:
Committed your suggestion